common/reviews/api/api-extractor.api.md

```diff
@@ -73,123 +104,109 @@ export const enum ExtractorMessageLogLevel {
 }
 
 // @public
-export const enum ExtractorValidationRulePolicy {
-    allow = "allow",
-    error = "error"
+export class ExtractorResult {
+    // @internal
+    constructor(properties: ExtractorResult);
+    readonly compilerState: CompilerState;
+    readonly errorCount: number;
+    readonly extractorConfig: ExtractorConfig;
+    readonly succeeded: boolean;
+    readonly warningCount: number;
```

Comment:
I expanded on it a bit in https://github.com/Microsoft/web-build-tools/issues/1133 but having an `apiChanged: boolean` property would be a useful result that can't easily be obtained from these other values.

Reply:
That's a good idea.  I'll add it in #1133 since this PR is already pretty big, and it's more relevant to that issue.